homeassistant/components/hvac/zwave.py

```diff
@@ -237,9 +238,9 @@ def set_swing_mode(self, swing_mode):
     @property
     def min_temp(self):
         """Return the minimum temperature."""
-        return self._convert_for_display(19)
+        return convert(19, TEMP_CELSIUS, self.unit_of_measurement)
```

Comment:
Why calculate the conversion every time? It seems computationally unnecessary, even if it is such an easy conversion. Would something like this be more efficient?

```python
@property
def min_temp(self):
    """Return the minimum temperature."""
    if 'F' in self.unit_of_measurement:
        return 66.2
    return 19
```

I might be missing something though.